gavel/templates/admin_annotator.html

```diff
@@ -28,6 +28,10 @@
         <td>{{ annotator.secret }}</td>
       </tr>
       <tr>
+        <th>Login URL</th>
+        <td><a href="{{ login_link }}" target="_blank">{{ login_link }}</a></td>
```

Comment:
Why `target="_blank"`? Imo overriding default behavior is kinda annoying. Thoughts?